packages/lesswrong/components/localGroups/CommunityHome.tsx

```diff
@@ -117,6 +115,7 @@ const CommunityHome = ({classes}: {
         <AnalyticsContext pageContext="communityHome">
           <Components.CommunityMapWrapper
             terms={mapEventTerms}
+            mapOptions={currentUserLocation.known && {center: currentUserLocation, zoom: 5}}
```

Comment:
We check for `known` so that we don't first zoom into the default location, which is Berkeley